operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -679,7 +679,11 @@ function wdtCreatePrimordialDomain() {
     > ${WDT_OUTPUT}
   ret=$?
   if [ $ret -ne 0 ]; then
-    trace SEVERE "WDT Create Domain Failed ${ret}"
+    if [ "JRF" == "$WDT_DOMAIN_TYPE" ] ; then
+      trace SEVERE "MII Fatal Error: WDT Create Domain Failed ${ret}"
+    else
```

Comment:
It looks like MII Fatal is used for _any_ JRF failure path, instead of just the JRF schema create path.  

I strongly suspects we still want retries for failures other than JRF schema create.

Reply:
There is no schema create path in WDT create domain,  this is a single entry point and unless there is a way WDT can differentiate it and return differently

Reply:
If there's no wallet file, then you know, right?

Reply:
If the wallet is wrong or passphrase is wrong, should we auto retry?  I am just on the safe side but we can try to open it up

Reply:
The update looks good to me, thanks.  Added a separate comment asking for more comments.